docs/content/smb.md

```diff
@@ -0,0 +1,231 @@
+---
+title: "SMB / CIFS"
+description: "Rclone docs for SMB backend"
+---
+
+# {{< icon "fa fa-server" >}} SMB
+
+SMB is [a communication protocol to share files over network](https://en.wikipedia.org/wiki/Server_Message_Block).
+
+This relies on [go-smb2 library](https://github.com/hirochachacha/go-smb2/) for communication with SMB protocol.
+
+Paths are specified as `remote:sharename` (or `remote:` for the `lsd`
+command.)  You may put subdirectories in too, e.g. `remote:item/path/to/dir`.
+
+## Notes
+
+The first path segment must be the name of the share, which you entered when you started to share on Windows. On smbd, it's the section title in `smb.conf` (usually in `/etc/samba/`) file.
+You can find shares by quering the root if you're unsure (e.g. `rclone lsd remote:`).
+
+You can't access to the shared printers from rclone, obviously.
+
+You can't use Anonymous access for logging in. You have to use the `guest` user with an empty password instead. (Setting empty password requires tweaking `rclone.conf` file)
+The rclone client tries to avoid 8.3 names when uploading files. If you wish to prevent it, it is highly recommended to use either [FTP](/ftp) or [SFTP](/sftp) instead.
```

Comment:
Have we done enough to prevent 8.3 names? How might they happen?

It would be nice to remove "If you wish to prevent it, it is highly recommended to use either [FTP](/ftp) or [SFTP](/sftp) instead." if we can

Reply:
The situation happens when the file name ends with spaces or dots (e.g. `rclone ` or `smb.go.`)
As SMB itself doesn't seem to have a way to convert it back to non-8.3 name, the encoding parameter has `EncodeRightSpace` and `EncodeRightPeriod`
It is the same for Windows - you'll see 8.3 names for these files too

Reply:
Ah, so we have fixed the problem with the encoding? I noticed this problem when I ran the tests without the encodings by accident.

Perhaps we should change the docs to

> The rclone client tries to avoid 8.3 names when uploading files by encoding trailing spaces and periods.

And drop the 

> If you wish to prevent it, it is highly recommended to use either [FTP](/ftp) or [SFTP](/sftp) instead.

Which I think will confuse users.